### PR TITLE
:seedling: e2e tests for cronjob

### PR DESCRIPTION
**Pull request by naveensrinivasan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
  Test for cronjob

* **What is the current behavior?** (You can also link to an open issue here)
 No tests for cron job

* **What is the new behavior (if this is a feature change)?**

  * Implemented basic e2e tests for cornjob

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
 None

* **Other information**:
None

### CODE REVIEW

File: e2e/cron-projects.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+repo,metadata
```
Comment (inferno-chromium) [inline e2e/cron-projects.txt:1]: add "# " at start ?

File: e2e/cron_test.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("E2E TEST:cron", func() {
+	Context("E2E TEST:Validating cron test", func() {
+		It("Should return valid test results for cron", func() {
+			fileName := fmt.Sprintf("%02d-%02d-%d.json",
```
Comment (inferno-chromium) [inline e2e/cron_test.go:29]: %04d instead of %d to be more exact with year :)

File: e2e/cron_test.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+				time.Now().Month(), time.Now().Day(), time.Now().Year())
+			_, err := ioutil.ReadFile("../" + fileName)
+			Expect(err).Should(BeNil())
+
+			// TODO - include validation to checks for file exists in the GCS bucket
```
Comment (inferno-chromium) [inline e2e/cron_test.go:34]: s/TODO -/TODO(naveensrinivasan): /

**Review by inferno-chromium — APPROVED** [review]

File: e2e/cron-projects.txt (hunk 1 above)
Comment (naveensrinivasan) [reply e2e/cron-projects.txt:1]: It is a CSV file. The code looks for a header.

File: e2e/cron_test.go (hunk 2 above)
Comment (naveensrinivasan) [reply e2e/cron_test.go:29]: Using the existing format to be consistent https://github.com/ossf/scorecard/blob/main/cron/main.go#L57

File: e2e/cron_test.go (hunk 3 above)
Comment (naveensrinivasan) [reply e2e/cron_test.go:34]: Why just naveensrinivasan 😉 ? 

Instead, I will create an issue and reference this PR.

**Pull request merged**